runtime/vm/threadhelp.cpp

```diff
@@ -419,4 +423,78 @@ failedToSetAttr(IDATA rc)
 	return ((rc != J9THREAD_SUCCESS) && (rc != J9THREAD_ERR_UNSUPPORTED_ATTR));
 }
 
+#if defined(J9VM_ZOS_3164_INTEROPERABILITY)
+void
+queryJNIEnv31(J9VMThread* vmThread)
```

Comment:
These functions need some kind of error handling, even if it's just an assert. The callers expect them to succeed.

Reply:
I considered ways to pass the success/failure, but not sure if there's a good way to handle the error on from the callers.  In any case, I expect this to succeed once we have the 31-bit shim library in place.  Added `VM_assert_true` to both functions.  Note, I updated the type of both `vmThread->jniEnv31` and `vm->javaVM31` to U_32 to ensure these 32-bit values are passed properly to the 31-bit targets (via ffi).   Hence, I updated the comparisons to be `0 !=` as opposed to `NULL !=`.  Fixed under: 1220675750c56aa141a7a8819cb60884da621a3f